fixtures/eurolite/led-bar-3-hcl-bar.json

```
@@ -0,0 +1,303 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "LED Bar-3 HCL Bar",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Rainer Jung"],
+    "createDate": "2020-01-13",
+    "lastModifyDate": "2020-01-13"
+  },
+  "links": {
+    "manual": [
+      "https://www.steinigke.de/download/51930395-Manual-110557-1.000-eurolite-led-bar-3-hcl-bar-de_en.pdf"
+    ],
+    "productPage": [
+      "https://www.steinigke.de/en/mpn51930395-eurolite-led-bar-3-hcl-bar.html"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=SV2AFTdJKNE"
+    ]
+  },
+  "physical": {
+    "dimensions": [265, 129, 87],
+    "weight": 1.13,
```

The manual mentions 0.9kg. Maybe the 1.13kg mentioned on the product page is measured with packaging? Or one value is just wrong :sweat_smile: 

```suggestion
    "weight": 0.9,
```